spearhead/assets/sass/navigation.scss

```diff
@@ -21,3 +35,25 @@ $navigation-max-break-point: 'laptop-only';
 		}
 	}
 }
+
+.social-navigation > div {
+	ul li {
+		width: auto;
+	}
+}
+
+.primary-navigation-open .extra-navigation-wrapper {
+	background: var(--primary-nav--color-background);
+	padding: var(--global--spacing-unit) var(--global--spacing-horizontal);
+	visibility: visible;
+	opacity: 1;
+	top: 50px;
+	right: 0;
+	bottom: 30px; // Not sure why this div is offset by 30px
```

Comment:
I believe this is offset by 30px because the navigation contains a translation that shifts it up by ~30px: 

https://github.com/Automattic/themes/blob/trunk/seedlet/assets/sass/components/navigation/_style.scss#L166